docs/development.md

```diff
@@ -229,38 +233,6 @@ then add
 
 to the main katello server you want the client attached to
 
-## Webpack
-
-Forklift creates a reverse proxy between localhost:3000 (Rails development default server) and https://forkliftfqdn (e.g: https://centos7-devel.example.com). This will cause mixed-content errors if you try to enable the webpack server without any options, as it needs to use the right certificates for HTTPS and Foreman needs to allow it. If you do not need to hack on Webpack code, ensure `:webpack_dev_server` is set to false in `~/foreman/config/settings.yaml` and run `bundle exec rake webpack:compile` from `~/foreman`.That will create the necessary assets to be served by Foreman. If you want to work on Webpack code, follow these instructions:
-
-To work with webpack-dev-server, ensure the following lines are in your `~/foreman/config/settings.yaml`:
-
-```yaml
-:webpack_dev_server: true
-:webpack_dev_server_https: true
-```
-
-This will instruct Foreman to use Webpack from an https source. Now we need to make sure Webpack is using the right certificates to serve the development content through HTTPS. We're going to save these configuration values on `~/foreman/.env`:
-
-```
-WEBPACK_OPTS='--https --key /etc/pki/katello/private/katello-default-ca.key --cert /etc/pki/katello/certs/katello-default-ca.crt --cacert /etc/pki/katello/certs/katello-default-ca.crt --host 0.0.0.0 --public centos7-devel.example.com'
```

Comment:
This is handled by puppet-katello_devel or just old information?

Reply:
This is now handled by puppet-katello_devel.